sunbird_mcp/src/api/sandbox_search/api.py

```
@@ -0,0 +1,237 @@
+        try:
+            async with self.session.post(
+                url,
+                json=payload,
+                headers={"Content-Type": "application/json"}
+            ) as response:
+                response.raise_for_status()
+                return await response.json()
+                
+        except aiohttp.ClientError as e:
+            logger.error("Sandbox Search API request failed for URL %s: %s", url, str(e), exc_info=True)
+            raise SunbirdAPIError(f"Failed to execute search in sandbox: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)